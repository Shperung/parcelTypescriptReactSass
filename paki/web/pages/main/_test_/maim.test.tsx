import React from 'react';
import {Provider} from 'react-redux';
import {Router} from 'react-router-dom';
import renderer from 'react-test-renderer';
import {createMemoryHistory} from 'history';

const history = createMemoryHistory('/');
import create from '../../../../redux/store.test';

import MainPage from '../main.page';

it('Main page', () => {
  const store = create();
  const snap = renderer
    .create(
      <Provider store={store}>
        <Router history={history}>
          <MainPage />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(snap).toMatchSnapshot();
});
