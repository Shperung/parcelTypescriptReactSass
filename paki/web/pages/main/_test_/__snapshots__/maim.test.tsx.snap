// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page 1`] = `
<section>
  <h1>
    MainPage 1
  </h1>
  <a
    href="/admin"
    onClick={[Function]}
  >
    admin2
  </a>
  <ul>
    0
  </ul>
</section>
`;
